### ISSUE TITLE
Color-coding priority/urgency

### ISSUE THREAD

**Issue by jcommelin** [garykl/task2dot#3]
Thanks for writing this code! It's really useful to get an overview of my flooded todo-list.

I was wondering, do you think it would make sense to color-code the urgency of a task?

More general question/idea: Can I somehow configure the color scheme of the resulting graph?

**Comment by garykl** [comment]
I'm glad you like it!
I just made an update to the repository and uploaded it to pypi, version 0.0.12. It is now possible to define custom colors within a configuration file. See the updated README.md at the bottom for details. Hope this helps.
Concerning the color code to urgency, I'm a bit hesitant to provide something similar to this. There is actually an ancestor version of task2dot, called dotwarrior (https://github.com/garykl/dotwarrior), which could do exactly this. I rewrote it to simplify some things and got rid of urgency colorization because it didn't really fit into my use case and the resulting graphs didn't really make graphs easier to recognize. If I can find I way to include it without messing up the code structure and configuration too much, I might add it later.
